html-practice/facebook-mock-up.html

```diff
@@ -0,0 +1,121 @@
+		</header>
+		<main id="content">
+			<div class="status-box">
+				<ul class="status-box-options">
+					<li><span>Update Status</span></li>
+					<li><span>Add Photos/Video</span></li>
+					<li><span>Create Photo Album</span></li>
+				</ul>
+				<div class="status-update-input">
+				<img src="logo.png"><input type="text" placeholder="What's on your mind?">
+				</div>
+				<div>
+					<ul class="status-update-options">
+						<li><span>Photo</span></li>
+						<li><span>Tag</span></li>
+						<li><span>Smile</span></li>
+						<li><span>Location</span></li>
+						<li><input type="button" id="status-privacy" value="Post"></li>
+						<li><input type="button" id="status-post" value="Friends"></li>
+					</ul>
+				</div>
+			</div>
+			<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. In id justo ac nisi feugiat sodales. Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia Curae; Sed vel odio nec velit pretium ultrices. Proin sagittis quam ligula, at tempor urna pretium eget. Curabitur mollis justo et mauris elementum, non tempus purus vulputate. Morbi vel lectus neque. Mauris convallis dui ipsum, at laoreet quam ultrices non. Mauris ut mi eget eros porta scelerisque. Nulla accumsan non purus vel dictum. Sed ut sem diam</p>
+			<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. In id justo ac nisi feugiat sodales. Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia Curae; Sed vel odio nec velit pretium ultrices. Proin sagittis quam ligula, at tempor urna pretium eget. Curabitur mollis justo et mauris elementum, non tempus purus vulputate. Morbi vel lectus neque. Mauris convallis dui ipsum, at laoreet quam ultrices non. Mauris ut mi eget eros porta scelerisque. Nulla accumsan non purus vel dictum. Sed ut sem diam</p>
+			<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. In id justo ac nisi feugiat sodales. Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia Curae; Sed vel odio nec velit pretium ultrices. Proin sagittis quam ligula, at tempor urna pretium eget. Curabitur mollis justo et mauris elementum, non tempus purus vulputate. Morbi vel lectus neque. Mauris convallis dui ipsum, at laoreet quam ultrices non. Mauris ut mi eget eros porta scelerisque. Nulla accumsan non purus vel dictum. Sed ut sem diam</p>
+			<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. In id justo ac nisi feugiat sodales. Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia Curae; Sed vel odio nec velit pretium ultrices. Proin sagittis quam ligula, at tempor urna pretium eget. Curabitur mollis justo et mauris elementum, non tempus purus vulputate. Morbi vel lectus neque. Mauris convallis dui ipsum, at laoreet quam ultrices non. Mauris ut mi eget eros porta scelerisque. Nulla accumsan non purus vel dictum. Sed ut sem diam</p>
+		</main>
+		<sidebar id="profile-content">
+		</sidebar>
```

Comment:
`<sidebar>`  is not a valid HTML 5 tag.  While there is a specification for  custom tags, it's not wise to use custom tags if you want to maintain support for older browsers.  Also, custom tags must include a `-`  character. See this stack overflow answer: http://stackoverflow.com/a/9845124